### DISCUSSION TITLE
Cached data, error status and pending promises in data fetching

### DISCUSSION THREAD

**Discussion by starter-dev** [nuxt/nuxt#19440] [Questions] [answered]
Does the cached data, error status, and pending promises of `useFetch` and `useAsyncData` get deleted automatically when the component where it is called is unmounted?

**Comment by TheAlexLichter** [comment, marked as answer]
It should not leak but data should be still available under the same key, e.g. via `useNuxtData` or similar.

**Reply by starter-dev** [reply]
Thank you so much for the reply!
